src/bin/gifski.rs

```diff
@@ -106,6 +106,11 @@ fn bin_main() -> BinResult<()> {
                             .empty_values(false)
                             .use_delimiter(false)
                             .required(true))
+                        .arg(Arg::with_name("loop-count")
+                            .long("loop-count")
+                            .help("Number of loops the animation is shown")
```

Comment:
```suggestion
                            .help("Number of times the animation is repeated")
```

The previous sentence doesn't read very well.